local_planner/src/nodes/planner_functions.cpp

```diff
@@ -32,9 +31,8 @@ void processPointcloud(pcl::PointCloud<pcl::PointXYZI>& final_cloud,
     for (const pcl::PointXYZ& xyz : cloud) {
       // Check if the point is invalid
       if (!std::isnan(xyz.x) && !std::isnan(xyz.y) && !std::isnan(xyz.z)) {
-        if (histogram_box.isPointWithinBox(xyz.x, xyz.y, xyz.z)) {
           distance = (position - toEigen(xyz)).norm();
-          if (distance > min_realsense_dist && distance < histogram_box.radius_) {
+          if (distance > min_realsense_dist ) {
```

Comment:
```suggestion
          if (distance > min_realsense_dist && distance < sensor_range ) {
```

And you need to bring in the sensor_range from the planner

Reply:
For the memory points - sure we need to make sure it doesn't blow up. But I don't see why we would arbitrarily limit the range of our sensor. This way, as far as you see, you have data. Why the hell not. Makes the algorithm sensor-independent. Ignoring points that are far away is done by the obstacle cost term in the cost function.